server/config.go

```diff
@@ -126,5 +127,6 @@ func NewConfig() *Config {
 		VisitorEmailLimitBurst:               DefaultVisitorEmailLimitBurst,
 		VisitorEmailLimitReplenish:           DefaultVisitorEmailLimitReplenish,
 		BehindProxy:                          false,
+		DisableWebUI:                         false,
```

Comment:
Can you name this `WebEnabled` instead and make it `true` by default? That will invert the `if` statements.